Diagnose-PrinterBlockTimeline.ps1

```
@@ -0,0 +1,193 @@
+    } catch { continue }
+    foreach ($line in $lines) {
+        # Only the meaningful apply markers, to avoid flooding the timeline.
+        if ($line -notmatch 'Apply task fired|Applying via|Apply succeeded|Apply failed|Hash matches|ImportReport') { continue }
+        $m = [regex]::Match($line, $tsRegex)
+        if (-not $m.Success) { continue }
+        $t = $null
+        if ([datetime]::TryParse($m.Groups[1].Value, [ref]$t)) {
+            if ($t -ge $since) {
+                $short = ($line -replace $tsRegex, '').Trim()
+                if ($short.Length -gt 60) { $short = $short.Substring(0,60) }
+                Add-Row $t 'HSS' (Split-Path $lf -Leaf)
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Add-Row' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)